editor/src/core/model/element-template-utils.ts

```diff
@@ -765,6 +744,18 @@ export function elementOnlyHasTextChildren(element: JSXElementChild): boolean {
       return allElementsAndChildrenAreText(element.children)
     case 'JSX_TEXT_BLOCK':
       return textBlockIsNonEmpty(element)
+    case 'ATTRIBUTE_VALUE':
+      return typeof element.value === 'string'
+    case 'ATTRIBUTE_NESTED_ARRAY':
+      return element.content.every((contentElement) => {
+        return elementOnlyHasTextChildren(contentElement.value)
+      })
+    case 'ATTRIBUTE_NESTED_OBJECT':
+      return element.content.every((contentElement) => {
+        return elementOnlyHasTextChildren(contentElement.value)
+      })
+    case 'ATTRIBUTE_FUNCTION_CALL':
+      return allElementsAndChildrenAreText(element.parameters)
     default:
```

Comment:
intriguing! does this have an effect on the text editing capabilites?

Reply:
Now removed with d80aab8e9.